src/ppx/grfx/dx12/dx12_swapchain.cpp

```diff
@@ -378,6 +397,75 @@ Result Swapchain::PresentInternal(
     return ppx::SUCCESS;
 }
 
+Result Swapchain::Resize(uint32_t width, uint32_t height)
+{
+    mCreateInfo.width  = width;
+    mCreateInfo.height = height;
+
+    std::vector<ID3D12Resource*> colorImages;
+    for (auto& im : mColorImages) {
+        auto pRes = static_cast<dx12::Image*>(im.Get())->GetDxResource();
+        colorImages.push_back(pRes);
+    }
+
+    colorImages[0]->AddRef();
```

Comment:
Are these AddRef and Release still needed? Here and on line 419.

Reply:
No. Removed.